projects/driver_controls/inc/turn_signal_fsm.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+// Monitors the current state of the turn signals (Left, Right, or None)
+
+#include "fsm.h"
+
+StatusCode turn_signal_fsm_init(FSM *turn_signal_fsm);
```

Comment:
This is the only FSM where you haven't called this variable `fsm`. Any reason? :P